src/features/embalm/stepContent/steps/TotalRequiredArchaeologists.tsx

```diff
@@ -61,14 +61,14 @@ export function TotalRequiredArchaegologists() {
         )}
       </Text>
       <Divider my={6} />
-      <Text>
-        How many archaeologists are
-        required to unwrap the sarcophagus?
-      </Text>
+      <Text>How many archaeologists are required to unwrap the sarcophagus?</Text>
       <FormControl>
         <Flex mt={6}>
           <Flex direction="column">
-            <NumberInput defaultValue={1} min={1} max={totalArchaeologists || 1}
+            <NumberInput
+              defaultValue={1}
```

Comment:
did we decide to default to 0 here? I vaguely remember hearing that on discord

Reply:
Note -- issue for this is here:
https://github.com/sarcophagus-org/sarcophagus-v2-app/issues/105